Tests/UnitTests/Core/Geometry/TrackingVolumeCreation.hpp

```diff
@@ -20,11 +20,17 @@
 namespace Acts {
 
 ///  helper function to create a cylinder
-TrackingVolumePtr constructCylinderVolume(
-    const GeometryContext& gctx, double surfaceHalfLengthZ, double surfaceR,
-    double surfaceRstagger, double surfaceZoverlap, double layerEnvelope,
-    double volumeEnvelope, double innerVolumeR, double outerVolumeR,
-    const std::string& name) {
+TrackingVolumePtr
+constructCylinderVolume(const GeometryContext& gctx,
+                        double surfaceHalfLengthZ,
+                        double surfaceR,
+                        double surfaceRstagger,
+                        double surfaceZoverlap,
+                        double layerEnvelope,
+                        double volumeEnvelope,
+                        double innerVolumeR,
+                        double outerVolumeR,
+                        const std::string& name) {
   ///  the surface transforms
```

Comment:
Another particularly horrific example

Reply:
Again, I don't know... the former I can't really read without playing compiler and mentally grepping for commas, the latter I can.

Reply:
Interesting. It’s almost the opposite for me. I guess I read more from left to right than from top to bottom.